linux/files/dt/avp64.dts

```diff
@@ -227,6 +227,28 @@
             };
         };
 
+        m_can1: can@10021000 {
```

Comment:
m_can0?

Reply:
Yes, you're right. It should be `m_can0`. I fixed it